sec2_the-ruby-programming-language/2.12_final_ruby_project/1_bcrypt.rb

```diff
@@ -0,0 +1,24 @@
+require 'bcrypt'
+
+my_password = BCrypt::Password.create("my password")
+
+puts my_password
+puts my_password.version              #=> "2a"
```

Comment:
What is this `.version`?

Reply:
This is a method by the Bcrypt library. Go find their library and. comment here what it does.

Reply:
`version` is an attribute in the module `BCrypt`, class `Password` where it shows the version of the bcrypt algorithm used to create a hash.
When `BCrypt::Password` is created (in Line 3), it initialise a hash and split the hash with `$` and get the version.